app/helpers/alchemy/admin/tags_helper.rb

```diff
@@ -18,7 +18,7 @@ def render_tag_list(class_name, params)
           li_s << content_tag('li', name: tag.name, class: tag_list_tag_active?(tag, params) ? 'active' : nil) do
             link_to(
               "#{tag.name} (#{tag.count})",
-              url_for(
+              resource_url_proxy.url_for(
```

Comment:
Ah my problem was I was including the `tags_helper` in a controller outside of alchemy. The `tags_helper` then tries to use the `alchemy` url proxy, but my routes are not in there.